Makefile

```diff
@@ -137,7 +135,7 @@ ifeq ($(SHOULD_MINIKUBE_MOUNT),1)
 endif
 	source ${HOME}/.virtualenvs/${VENV_NAME}/bin/activate && \
 	minikube docker-env --profile ${MINIKUBE_PROFILE} > /dev/null && eval $$(minikube docker-env --profile ${MINIKUBE_PROFILE}) && \
-	reana-cluster -f ${PWD}/../reana-cluster/reana_cluster/configurations/reana-cluster-minikube$(addprefix -, ${CLUSTER_CONFIG}).yaml $(CLUSTER_FLAGS) down --delete-traefik --delete-secrets && \
+	helm dep update helm/reana-stable && helm uninstall reana 2> /dev/null || \
```

Comment:
Let's not redirect to `/dev/null` to be safe... Either add some `if $(helm ls) ...` condition, or else just let the output be printed (since `make` is only used by developers, it'll do).